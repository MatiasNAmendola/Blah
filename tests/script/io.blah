writeLine(isEOF())
firstLine: readLine()
writeLine(isEOF())
in: readLine
secondLine: in()
write(isEOF())
write(',')
write(readLine())
write(' ')
out: write
out(Nothing)
writeLine('|')
out(13, -54)
write('\n')
write(True, False)
writeLine()
writeLine(1, firstLine)
writeLine(2 & ' ' & secondLine)
