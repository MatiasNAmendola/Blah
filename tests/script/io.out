False
False
True,Nothing Nothing|
13 -54
True False
1 hi
2 hello
